### ISSUE TITLE
Problem with setup during pip install

### ISSUE THREAD

**Issue by RJacobArthrex** [ZFTurbo/segmentation_models_pytorch_3d#6]
### Issue found on:
Python Version: 3.10
OS: Linux

### Description:
After `pip install segmentation-models-pytorch-3d`, when I try to run the example code, following error occurs:

```python
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Cell In[1], line 1
----> 1 import segmentation_models_pytorch_3d as smp
      2 import torch
      4 model = smp.Unet(
      5     encoder_name="efficientnet-b0", # choose encoder, e.g. resnet34
      6     in_channels=1,                  # model input channels (1 for gray-scale volumes, 3 for RGB, etc.)
      7     classes=3,                      # model output channels (number of classes in your dataset)
      8 )

File /opt/conda/lib/python3.10/site-packages/segmentation_models_pytorch_3d/__init__.py:7
      4 from . import losses
      5 from . import metrics
----> 7 from .decoders.unet import Unet
      8 from .decoders.unetplusplus import UnetPlusPlus
      9 from .decoders.manet import MAnet

ModuleNotFoundError: No module named 'segmentation_models_pytorch_3d.decoders.unet'
```

### Reason for error:
This is due to `setup.py` where only first level packages are given:
```python
    packages=[
        'segmentation_models_pytorch_3d',
        'segmentation_models_pytorch_3d/base',
        'segmentation_models_pytorch_3d/datasets',
        'segmentation_models_pytorch_3d/decoders',
        'segmentation_models_pytorch_3d/encoders',
        'segmentation_models_pytorch_3d/losses',
        'segmentation_models_pytorch_3d/metrics',
        'segmentation_models_pytorch_3d/utils',
    ],
```
If I import `find_packages` from `setuptools`, I get that these should be all the packages that need to be listed:
```python
['segmentation_models_pytorch_3d',
 'segmentation_models_pytorch_3d.losses',
 'segmentation_models_pytorch_3d.datasets',
 'segmentation_models_pytorch_3d.base',
 'segmentation_models_pytorch_3d.encoders',
 'segmentation_models_pytorch_3d.decoders',
 'segmentation_models_pytorch_3d.utils',
 'segmentation_models_pytorch_3d.metrics',
 'segmentation_models_pytorch_3d.decoders.linknet',
 'segmentation_models_pytorch_3d.decoders.unet',
 'segmentation_models_pytorch_3d.decoders.deeplabv3',
 'segmentation_models_pytorch_3d.decoders.pan',
 'segmentation_models_pytorch_3d.decoders.pspnet',
 'segmentation_models_pytorch_3d.decoders.fpn',
 'segmentation_models_pytorch_3d.decoders.unetplusplus',
 'segmentation_models_pytorch_3d.decoders.manet']
```

### Solution:
Either update the code in `setup.py` code to use `find_packages` like given in https://setuptools.pypa.io/en/latest/userguide/quickstart.html#package-discovery or add the above packages to the existing list.

**Comment by RJacobArthrex** [comment]
TL;DR version of above is `setup.py` should be changed to:
```python
from setuptools import setup

setup(
    name='segmentation_models_pytorch_3d',
    version='1.0.0',
    author='Roman Sol (ZFTurbo)',
    packages=[
        'segmentation_models_pytorch_3d',
        'segmentation_models_pytorch_3d/losses',
        'segmentation_models_pytorch_3d/datasets',
        'segmentation_models_pytorch_3d/base',
        'segmentation_models_pytorch_3d/encoders',
        'segmentation_models_pytorch_3d/decoders',
        'segmentation_models_pytorch_3d/utils',
        'segmentation_models_pytorch_3d/metrics',
        'segmentation_models_pytorch_3d/decoders/linknet',
        'segmentation_models_pytorch_3d/decoders/unet',
        'segmentation_models_pytorch_3d/decoders/deeplabv3',
        'segmentation_models_pytorch_3d/decoders/pan',
        'segmentation_models_pytorch_3d/decoders/pspnet',
        'segmentation_models_pytorch_3d/decoders/fpn',
        'segmentation_models_pytorch_3d/decoders/unetplusplus',
        'segmentation_models_pytorch_3d/decoders/manet',
    ],
    url='https://github.com/ZFTurbo/segmentation_models_pytorch_3d',
    description='Set of models for segmentation of 3D volumes using PyTorch.',
    long_description='3D variants of popular models for segmentation like FPN, Unet, Linknet etc using Pytorch module.'
                     'Automatic conversion of 2D imagenet weights to 3D variant',
    install_requires=[
        'torch',
        'torchvision>=0.5.0',
        "pretrainedmodels==0.7.4",
        "efficientnet-pytorch==0.7.1",
        "timm==0.9.7",
        "tqdm",
        "pillow",
        "six",
    ],
)
```

**Comment by ZFTurbo** [comment]
Hello. You are right. I did poposed changes in setup.py and create new release 1.0.1 - it must work ok now. Can you please check?

**Comment by RJacobArthrex** [comment]
Thank you for the fix. Confirmed that version 1.0.1 fixed this problem. Marking this issue as closed.
